account_banking_payment_transfer/model/account_payment.py

```diff
@@ -253,3 +268,22 @@ def action_sent(self):
         # State field is written by act_sent_wait
         self.write({'date_sent': fields.Date.context_today(self)})
         return True
+
+    @api.multi
+    def partial(self):
+        self.ensure_one()
+        view_id = self.env.ref('account.view_move_line_tree').id
```

Comment:
Don't you want a custom view here instead of taking the one from the account module? I say that because of the issue produced by on_create_write on view_move_line_tree. The domain that you set in reconcile_partial_domain will be replaced by the domain defined in on_create_write. Let me know if I am not clear, else, I described the problem a bit further here: https://github.com/OCA/bank-payment/issues/93
Or do you think of an other solution?